neat_internal.h

```diff
@@ -30,6 +30,14 @@
 
 #define NEAT_MAX_NUM_PROTO 4
 
+#if defined(USRSCTP_SUPPORT) || defined(HAVE_NETINET_SCTP_H)
+#define SCTP_SUPPORTED
+#else
+#if defined(__linux__) // Need a further test for Linux SCTP support
+#define SCTP_SUPPORTED
```

Comment:
OK. I would assume that if someone wants to use SCTP, he has the proper headers installed.
I haven't understood why Linux makes headers an optional install although they provide the functionality always. One problem resulting from this is that the kernel headers used for compiling the kernel might be different from the headers the application sees. Have been running into this in the past. Never understood it.